### ISSUE TITLE
show the name of the chord

### ISSUE THREAD

**Issue by sugizo** [mikehadlow/gtr-cof#42]
it's nice to have show the name of the chord when user click the note e.g.
c-e-g = C Mayor
a-c-e = A Minor
etc

thanks

**Comment by mikehadlow** [comment]
I think it's a very nice idea to spell out the notes and intervals of the chord like this:
C-E-G  1-M3-P5
Also easy to implement since the algorithm already knows this information. 
Actually naming the chord is much trickier, especially since any chord can be named in several different ways with inversions

**Comment by sugizo** [comment]
yeah, when you talk about inversion, must discussed further because same note, different root note can have different chord name e.g.
c-d-g (csus2)
g-c-d (gsus4)

c-e-g-a (cmaj6)
a-c-e-g (am7)

c-e-g-b (cmaj7)
e-g-b-c (emb6)

**Comment by sugizo** [comment]
there's hundred combination that can be achieved to form a key, not to say that some of chords have a limit with chromatic (12 semitones), e.g. extended chord (9th, 11th, 13th chords that have more than 12 semitones)
perhaps to stay focus with your current work related with chord, best to add the relative key. provided the reliable source what can do with circle of fifth and the relative key, and nice to have it on guitar dashboard.

**ref**
https://en.wikipedia.org/wiki/Relative_key

**Comment by amiika** [comment]
Maybe there could be some sort of dropbox that would select correct intervals from the chord type, like:

maj  = I M3 5
7 = I M3 5 M7
min = I m3 5
...

**Comment by mikehadlow** [comment]
Note, this is a duplicate of #28

**Comment by mikehadlow** [comment]
Woops! closed the wrong issue by accident.
